tensorflow_addons/metrics/tests/streaming_correlations_test.py

```diff
@@ -128,7 +132,7 @@ def test_keras_binary_classification_model(self, correlation_type):
         tf.function(metric.update_state)(y, preds)
         metric_value = tf.function(metric.result)()
         scipy_value = self.scipy_corr[correlation_type](preds[:, 0], y[:, 0])[0]
-        np.testing.assert_almost_equal(metric_value, metric_history[-1])
+        np.testing.assert_almost_equal(metric_value, metric_history[-1], decimal=5)
```

Comment:
I had already this https://github.com/tensorflow/addons/pull/2782/commits/49a56e51f034a80f74e47c22f2ddb16945cdbf10